### PR TITLE
gh-115403: Remove extra colon after "Examples" in datetime documentation

### PR DESCRIPTION
**Pull request by wallseat**
* Issue: gh-115403

----
📚 Documentation preview 📚: https://cpython-previews--115452.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/datetime.rst [hunk 1]
```diff
@@ -1813,8 +1813,6 @@ Other constructor:
 
    Examples::
```
Comment (AlexWaygood) [inline Doc/library/datetime.rst]: Thanks! However, if we remove the `.. doctest::` line, then the doctest won't be run as part of our CI. I think the better fix is to remove the second colon after "Examples" -- something like this?

```suggestion
   Examples:

   .. doctest::

```

Comment (AlexWaygood) [reply Doc/library/datetime.rst]: (If you like my suggestion above, you can click the "commit suggestion" button immediately beneath the comment to apply the suggestion to your PR branch :)

Comment (wallseat) [reply Doc/library/datetime.rst]: Oh, sorry, my bad
i'll fix it

Comment (AlexWaygood) [reply Doc/library/datetime.rst]: No need to apologise!

**Review by AlexWaygood — APPROVED** [review]
LGTM, thanks!

**Comment by hugovk** [comment]
@wallseat Thank you and congratulations on your first CPython PR! 🎈 :rocket:

**Pull request merged**